helm/alfresco-content-services/values.yaml

```diff
@@ -122,7 +122,9 @@ repository:
 apiexplorer:
   ingress:
     path: /api-explorer
-ooiService:
+# -- Enable/Disable Alfresco Collaboration Connector for Microsoft 365
+alfresco-connector-ms365:
+  enabled: false
```

Comment:
see the ugly helm-docs generated, you want to move the comment under enabled

we also need to bump the doc where we declare upgrades with non backward compatible changes